translations/translations.json

```diff
@@ -3,6 +3,7 @@
     "de": {
       "createPdf": "PDF erstellen",
       "copyUrl": "Link kopieren",
+      "copyUrlTooltip": "Kopiert!",
```

Comment:
```suggestion
      "copied": "Kopiert!",
```

Reply:
Optional: We also have one `Kopiert` translation already, maybe we can reuse that one.